test/deploy/rackhd_stack_init.py

```diff
@@ -31,7 +31,20 @@
 MAX_CYCLES = 60
 
 class rackhd_stack_init(unittest.TestCase):
-    def test01_preload_sku_packs(self):
+    def test01_set_auth_user(self):
+        print '**** Installing default admin user'
+        fit_common.remote_shell('rm auth.json')
+        auth_json = open('auth.json', 'w')
+        auth_json.write('{"username":"' + fit_common.fitcreds()["api"][0]["admin_user"] + '", "password":"' \
+                        + fit_common.fitcreds()["api"][0]["admin_pass"] + '", "role":"Administrator"}')
+        auth_json.close()
+        fit_common.scp_file_to_ora('auth.json')
+        rc = fit_common.remote_shell("curl -ks -X POST -H 'Content-Type:application/json' https://localhost:" \
+                                     + str(fit_common.fitports()['https']) + "/api/2.0/users -d @auth.json")
```

Comment:
Just a suggestion:  I've seen this pattern a few times throughout the test code of creating a json file for the payload, scp-ing it to the ora and then running an API with the payload.  This would be a good candidate for a common function that takes a payload and a URL, and returns the result.